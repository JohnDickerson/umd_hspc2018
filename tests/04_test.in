10
1
2
3
100
1000
10000
100000
1000000
2500000
10000000
