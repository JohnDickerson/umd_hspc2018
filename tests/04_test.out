2
3
5
541
7919
104729
1299709
15485863
41161739
179424673
